### DISCUSSION TITLE
errore "Error while loading page config."

### DISCUSSION THREAD

**Discussion by roccia4694** [indomus/forum#208] [HUB: Home Assistant]
ciao a tutti è da oggi che su ha mi esce questo errore solamente quando provo ad aprire la tab impostazioni, ho provato a cancellare i cookie ed anche a riavviare il servizio su raspberry, oltre che ad avviare il raspberry, ma non riesco a risolvere; ho home assistant (ultima versione) core installato su venv e non ho fatto nessun aggiornamento o implementazione da quando ho iniziato a riscontrare il problema. Qualcun altro ha questo problema?

**Comment by snakuzzo** [comment]
No...credo nessuno.
Controlla cosa viene scritto sul log (`home-assistant.log`) quando ti capita

**Reply by roccia4694** [reply]
Ciao, grazie per la risposta, ho controllato il log ma questi sono gli unici errori che riscontro 
`2021-04-24 20:18:59 ERROR (MainThread) [homeassistant.components.netatmo] Error during webhook registration - 400 - Bad request - Allowed ports for webhooks are 80, 88, 443 and 9443. (21) when accessing 'https://api.netatmo.com/api/addwebhook'
2021-04-24 20:19:02 ERROR (MainThread) [haffmpeg.core] FFmpeg fails [Errno 2] No such file or directory: 'ffmpeg'
2021-04-25 02:05:46 ERROR (MainThread) [aiohttp.server] Unhandled exception
2021-04-25 02:05:46 ERROR (MainThread) [aiohttp.server] Unhandled exception
2021-04-25 02:05:46 ERROR (MainThread) [aiohttp.server] Unhandled exception
2021-04-25 02:05:48 ERROR (MainThread) [aiohttp.server] Unhandled exception
`

**Reply by snakuzzo** [reply]
Su quello di netatatmo forse può dirti di più @scialla... se non sbaglio per evitare quell'errore va esposta l'istanza HA su una di quelle porte (ma non vorrei dire cavolate).
Per quanto riguarda FFmpeg... che tipo di installazione hai ?

**Reply by roccia4694** [reply]
Ho HA core installato su venv; però da questi errori che mi escono non mi spiego ancora il malfunzionamento della tab impostazioni

**Reply by Scialla** [reply]
Per Netatmo, te lo puoi tenere così, puoi provare ad aprirglli una di quelle porte, ma a me non è cambiato nulla, continua a dare l'errore... In teoria serve per far andare netatmo tramite webhook ed avere un ritorno dello stato più realistico; in sostanza quando fai un settaggio la UI ci mette un pò ad aggiornarsi, aprendo le porte e abilitando webhook è molto più rapido, ma a molti non funziona. L'alternativa è integrare le valvole tramite homekit controller, diventa più reattivo. Di contro le entità create così sono meno e hai meno possibilità di configurare le valvole. Puoi anche integrare in entrambi i modi avendo però il doppio dei dispositivi.
